src/components/transactions/transactions.component.js

```diff
@@ -39,6 +41,15 @@ const TransactionsConstructor = function ($rootScope, $stateParams, $state, $htt
 		});
 	};
 
+	vm.makePages = (page) => {
+		const arr = [];
+		const n = Number(page);
+		if (n === 0 || n === 1 || n === 2) {
```

Comment:
Wouldn't be `n > 2` a better solution here?
The most common case should come first, exceptions later.